Formula/r/redis_exporter.rb

```diff
@@ -0,0 +1,60 @@
+class RedisExporter < Formula
+  desc "Prometheus Exporter for ValKey & Redis Metrics"
+  homepage "https://github.com/oliver006/redis_exporter"
+  url "https://github.com/oliver006/redis_exporter/archive/refs/tags/v1.71.0.tar.gz"
+  sha256 "5cc761bfe45c5414d21fc06a6a9081285b4811f5ce7be0d030cf135ad731211f"
+  license "MIT"
+
+  depends_on "go" => :build
+  depends_on "valkey" => :test
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w")
+
+    (bin/"redis_exporter_brew_services").write <<~SHELL
+      #!/bin/bash
+      exec #{bin}/redis_exporter $(<#{etc}/redis_exporter.args)
+    SHELL
+
+    touch etc/"redis_exporter.args"
```

Comment:
This isn't needed, people can just pass arguments themselves.

Reply:
Does Homebrew handle user-modified launchd plists gracefully? Do changes survive `brew services stop` + `brew services start`?

Reply:
Definitely not user modified, but if you create your own homebrew won't interfere.